workbench-core/example/infrastructure/integration-tests/tests/users/createUser.test.ts

```diff
@@ -20,7 +19,7 @@ describe('userManagement create user integration test', () => {
     user = {
       firstName: 'Test',
       lastName: 'User',
-      email: 'fakeemail@notanemail.com'
+      email: 'success+create-user@simulator.amazonses.com'
```

Comment:
Be careful with the static email address. If your test ever fails to clean up, any test which calls create() later is going fail because the user already exists. Either throw in some short random string, or double check that the user doesn't already exist before you begin.

Reply:
done!